tests/python/contrib/test_tedd.py

```diff
@@ -97,39 +103,41 @@ def verify():
 
 
 def test_schedule_tree():
+    """Tests schedule tree"""
     block_x = te.thread_axis("blockIdx.x")
     thread_x = te.thread_axis("threadIdx.x")
     n = te.var("n")
     m = te.var("m")
-    l = te.var("l")
-    A = te.placeholder((n, m, l), name="A")
-    B = te.compute((n, m, l), lambda bi, bj, bk: A[bi, bj, bk] + 1, name="B")
-    r = te.reduce_axis((0, m), "r")
-    C = te.compute(
+    op_l = te.var("op_l")
```

Comment:
IMO, it's good to keep those concise names. I guess these names can pass the pylint through the `pylintrc`.

Reply:
Thanks. I will change them later.

Reply:
@Hzfengsy Well, those concise names would trigger C0103(invalid-name). Like the var "l", the info is `Variable name "l" doesn't conform to '[a-z_][a-z0-9_]{2,30}$' pattern`. Is it better to use `pylint: disable=invalid-name` ?

Reply:
Sorry for the late reply, please feel free to disable invalid-name for the whole file

Reply:
Okay, got it,